src/test/librados/io_cxx.cc

```diff
@@ -462,6 +462,19 @@ TEST_F(LibRadosIoPP, XattrListPP) {
   }
 }
 
+TEST_F(LibRadosIoPP, CrcZeroWrite) {
+  char buf[128];
+  memset(buf, 0xcc, sizeof(buf));
+  bufferlist bl;
+
+  ASSERT_EQ(0, ioctx.write("foo", bl, 0, 0));
+  ASSERT_EQ(0, ioctx.write("foo", bl, 0, sizeof(buf)));
```

Comment:
Hmm, the only usage of the `0xcc`-filled `buf` seems to be the `sizeof()`. Is that truly the intention?

Reply:
@rzarzynski yes, i had in mind to check that we didn't had any override above zero, but i didn't added it. i can remove the memset.

Reply:
@rzarzynski please review